net.krafting.PedantiK.yml

```diff
@@ -0,0 +1,82 @@
+app-id: net.krafting.PedantiK
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
47 is out.

Reply:
True, I'll test it to see it it break anything